charts/trino/templates/deployment-coordinator.yaml

```diff
@@ -40,6 +40,9 @@ spec:
         runAsUser: {{ .runAsUser }}
         runAsGroup: {{ .runAsGroup }}
       {{- end }}
+      {{- if .Values.shareProcessNamespace.coordinator }}
```

Comment:
why conditionally?

Reply:
To not force user to define `shareProcessNamespace`